### ISSUE TITLE
Build is not possible with openJDK 21

### ISSUE THREAD

**Issue by N0N0** [takari/concurrent-build-logger#8]
`mvn clean integration-test -Pmaven-distro` does result in

```
6074 [ThreadedStreamConsumer] [INFO] Running io.takari.maven.logging.internal.LoggingIT
8344 [ThreadedStreamConsumer] [ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.262 s <<< FAILURE! - in io.takari.maven.logging.internal.LoggingIT
8344 [ThreadedStreamConsumer] [ERROR] io.takari.maven.logging.internal.LoggingIT.testBasic[target/maven-distro]  Time elapsed: 2.218 s  <<< FAILURE!
java.lang.AssertionError: Log line not present: I [test-project@main] slf4j-bridges jcl log message
        at org.junit.Assert.fail(Assert.java:89)
        at io.takari.maven.testing.executor.MavenExecutionResult.assertLogText(MavenExecutionResult.java:54)
        at io.takari.maven.logging.internal.LoggingIT.testBasic(LoggingIT.java:41)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
        at java.base/java.lang.reflect.Method.invoke(Method.java:580)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at io.takari.maven.testing.TestResources$1.evaluate(TestResources.java:34)
        at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
        at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
        at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
        at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
        at org.junit.runners.Suite.runChild(Suite.java:128)
        at org.junit.runners.Suite.runChild(Suite.java:27)
        at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
        at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
        at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:377)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:284)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:248)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:167)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
        at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)

8401 [main] [INFO]
8401 [main] [INFO] Results:
8401 [main] [INFO]
8401 [main] [ERROR] Failures:
8402 [main] [ERROR]   LoggingIT.testBasic:41 Log line not present: I [test-project@main] slf4j-bridges jcl log message
8402 [main] [INFO]
8402 [main] [ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
8402 [main] [INFO]
8404 [main] [INFO] ------------------------------------------------------------------------
8405 [main] [INFO] BUILD FAILURE
8405 [main] [INFO] ------------------------------------------------------------------------
8405 [main] [INFO] Total time:  7.543 s
8405 [main] [INFO] Finished at: 2023-10-27T09:08:46+02:00
8406 [main] [INFO] ------------------------------------------------------------------------
8406 [main] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M7:test (integration-test) on project concurrent-build-logger: There are test failures.
8406 [main] [ERROR]
8407 [main] [ERROR] Please refer to C:\concurrent-build-logger\target\surefire-reports for the individual test results.
8407 [main] [ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
```

Build is still possible with openJDK 17 though.

**Comment by N0N0** [comment]
Maven < 3.6.3 is not able to execute the enforcer and maven from version 3.6.3 on is giving the same error message.

**Comment by cstamas** [comment]
Fixed with c4db845b70e813f28d6f1f0ac4a80dbb8c448fe7
